apps/crn-server/setup-jest.ts

```diff
@@ -0,0 +1,13 @@
+import * as contentful from '@asap-hub/contentful';
+import { getContentfulEnvironmentMock } from './test/mocks/contentful-rest-client.mock';
+
+jest.mock('@asap-hub/contentful', () => {
```

Comment:
For me it seems better to mock globally because the alternative is to mock in a **LOT** of different files

Most of the tests that failed here https://github.com/yldio/asap-hub/actions/runs/4405132410/jobs/7715680051 are due to this function not being mock 

This seems the way to override the mock :)
https://stackoverflow.com/questions/55900737/how-to-override-jest-mock-defined-in-a-setupfiles-jest-config-js

Reply:
sorry but i have to disagree. I think mocking globally will introduce headaches. The reason we opt for very simple DI in BE is that we are able to track what is being called and where, at least in principle. You seem to follow that pattern and inject dependencies via constructor functions but then for some reason insist to just jest.mock and jest.spy which is not necessary with such simple DI. I still stand by what I said - i think this should go, even if you were going to just jest.mock and jest.spy in some places it should be in the file it is needed. 

Best if we ask the rest of team members @lennym @Riccardo-Brown